### PR TITLE
Add new buildbots

### PR DESCRIPTION
**Pull request by steven-johnson**
linux-1 and linux-4 are being configured; this adds them, but only for LLVM nightlies, so that halide testbranches aren't affected while I do some testing.

### CODE REVIEW

File: master/master.cfg [hunk 1]
```diff
@@ -66,8 +66,10 @@ c = BuildmasterConfig = {}
 password = Path('halide_bb_pass.txt').read_text().strip()
 
 _WORKERS = [
+    ('linux-worker-1', 8),
```
Comment (abadams) [inline master/master.cfg:69]: max builds of 8 seems like a lot. Wouldn't 1 be safer while getting these set up?

Comment (steven-johnson) [reply master/master.cfg:69]: I don't think so, since the initial tests will be either the nightlies, or me firing off force-builds.

**Comment by alexreinking** [comment]
Looks good to me, and it looks like CI likes it too!

**Review by alexreinking — APPROVED** [review]

File: master/master.cfg (hunk 1 above)
Comment (abadams) [reply master/master.cfg:69]: I just can't imagine any situation where doing 8 builds at once is safe. The linking step for llvm is liable to use many gigabytes of ram, and if those all occur at the same time...

Comment (steven-johnson) [reply master/master.cfg:69]: Fair. I'll revisit it.

**Pull request merged**